config/rbac/role.yaml

```diff
@@ -19,33 +18,31 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - services
+  - secrets
```

Comment:
Correct me if I'm wrong, but it looks like the only change here is to remove "batch/jobs" from the roles correct?

Reply:
That's correct. Things just got reordered when I reran the code generation.